src/modifier/modifier-type.ts

```diff
@@ -1868,7 +1868,7 @@ const modifierPool: ModifierPool = {
     new WeightedModifierType(modifierTypes.BATON, 2),
     new WeightedModifierType(modifierTypes.SOUL_DEW, 7),
     //new WeightedModifierType(modifierTypes.OVAL_CHARM, 6),
-    new WeightedModifierType(modifierTypes.CATCHING_CHARM, () => globalScene.gameMode.isDaily || (!globalScene.gameMode.isFreshStartChallenge() && globalScene.gameData.getSpeciesCount(d => !!d.caughtAttr) > 100) ? 4 : 0, 4),
+    new WeightedModifierType(modifierTypes.CATCHING_CHARM, () => globalScene.gameMode.isDaily || (!globalScene.gameMode.isClassic && globalScene.gameData.getSpeciesCount(d => !!d.caughtAttr) > 100) ? 4 : 0, 4),
```

Comment:
```suggestion
    new WeightedModifierType(modifierTypes.CATCHING_CHARM, () => !globalScene.gameMode.isClassic ? 4 : 0, 4),
```
I don't think the species count check is super necessary for Endless, the player *probably* has 100 mons by the time they beat Classic and unlock Endless, and even if they don't, probably not a big deal.

Reply:
Why be inconsistent across the different game modes? 
If we want to do this change, then we need to restructure the conditions for adding modifiers to the game.
Taking on this suggestion is adding code debt.

Reply:
Why be inconsistent across the different game modes? 
If we want to do this change, then we need to restructure the conditions for adding modifiers to the game.
Taking on this suggestion is adding code debt.